src/test/java/seedu/address/testutil/TypicalSessions.java

```diff
@@ -24,12 +26,14 @@
     public static final Session GETWELL = new SessionBuilder()
             .withGym(VALID_GYM_GETWELL)
             .withExerciseType(VALID_EXERCISE_TYPE_GETWELL)
-            .withInterval(VALID_START_TIME_GETWELL, VALID_DURATION_GETWELL)
+            .withInterval(LocalDateTime.parse(VALID_START_TIME_GETWELL, Interval.DATE_TIME_FORMATTER),
```

Comment:
[discussion] Is it better to do the new way or the previous way? 

cc @maguireong @dhafinrazaq

Reply:
I think both ways work but when I was doing it the other time, I didn't think much into it so I just put in the respective params into the method.

Reply:
in terms of consistency with the other with__ methods, should we pass the String? What do you all think?